README.md

```diff
@@ -199,6 +208,64 @@ tests for that API. For the pipeline to run those tests, do the following:
 
 The API tests run after smoke tests.
 
+### Opinionated infrastructure pipeline
+
+For infrastructure-only repositories e.g. "shared infrastructure" the library provides an opinionated infrastructure pipeline which will build Terraform files in the root of the repository.
+
+It uses a similar branch --> environment strategy as the app pipeline but with some differences for PRs
+
+Branch | Environment
+--- | ---
+`master` | `aat` then `prod`
+`demo` | `demo`
+`perftest` | `perftest`
+PR branch| `aat` (plan only)
+
+
+Example `Jenkinsfile` to use the opinionated infrastructure pipeline:
+```groovy
+#!groovy
+
+@Library("Infrastructure") _
+
+def product = "rhubarb"
+
+withInfraPipeline(product) {
+}
+```
+
+#### Slack notifications on failure / fixed
+
+To enable slack notifications when the build fails or is fixed add the following:
+```groovy
+withInfraPipeline(product) {
+  enableSlackNotifications('#my-team-builds')
```

Comment:
is it still needed to keep slack notifications in a separate section?
I would always want them enabled, maybe we could remove this block and add it to the main docs??

Reply:
Yeah was thinking similar - i'll extract out